app/src/main/java/com/example/weathertimeapp/mvp/contracts/WeatherDetailsContract.kt

```diff
@@ -1,15 +1,21 @@
 package com.example.weathertimeapp.mvp.contracts
 
-import com.example.weathertimeapp.data.entities.Forecast
+import com.example.weathertimeapp.data.entities.Day
 import io.reactivex.Observable
 
 interface WeatherDetailsContract {
     interface Model {
+        fun getExtendedWeather(cityData: Pair<Int?, Int?>): Observable<Day>
```

Comment:
Are you sure that the parameters of the Pair class can to be null?

Reply:
Fixed, an error when adding it led me to that but now its ok, thanks for the comment, i missed it

Reply:
no problem Bro